integration-tests/s3_test.go

```diff
@@ -76,13 +76,18 @@ var _ = Describe("S3", Label("s3"), func() {
 			instanceID, err := broker.Provision(s3ServiceName, "private", nil)
 
 			Expect(err).NotTo(HaveOccurred())
-			Expect(mockTerraform.FirstTerraformInvocationVars()).To(
+			vars, err := mockTerraform.FirstTerraformInvocationVars()
+			Expect(vars, err).To(
```

Comment:
This is a little unusual. It might be cleaner to test the error and value separately

Reply:
Hey @blgm it is an outdated line of an involuntary change, the content has remained as it was originally. Thank you very much